code/modules/mob/living/living.dm

```diff
@@ -620,6 +641,7 @@
 	if(!override)
 		float(!has_gravity)
 
+//TODO: Better floating
```

Comment:
can you not add these? they very rarely get followed up on, keep a todo list in your own management tool of choice